go.mod

```diff
@@ -0,0 +1,27 @@
+module github.com/openshift/cluster-api-provider-packet
```

Comment:
I think the module name in go.mod needs to be updated before I can import this.

Reply:
Latest changes update this.